### ISSUE TITLE
new check: suggest to iterate over dict.values() instead if items() if 

### ISSUE THREAD

**Issue by spaceone** [astral-sh/ruff#2206]
B007 unused-loop-control-variable revealed and fixed code like:

```
-               for k, v in self.validation_errors.items():
+               for _k, v in self.validation_errors.items():
                        num += v.number_of_errors()
```

a new checker could make it even better:
```
-               for k, v in self.validation_errors.items():
+               for v in self.validation_errors.values():
                        num += v.number_of_errors()
```

**Comment by andersk** [comment]
Some classes such as [`xml.etree.elementtree.Element`](https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.Element) support `items` but not `values`.

**Comment by JonathanPlasse** [comment]
It would be an aggressive lint.

**Comment by dhruvmanila** [comment]
This is being implemented as part of `perflint` (#4789)

**Comment by charliermarsh** [comment]
I believe we now support this via `PERF102`.
